docs/developers-clear-containers-install.md

```diff
@@ -76,38 +76,26 @@ For more details on the runtime's build system, run:
 $ make help
 ```
 
-4. Qemu-Lite
+4. Qemu-Lite, clear-containers image and kernel
```

Comment:
@iphutch fixed, as these are 3 different components that we are installing, I kept the comma. Please let me know if that is ok.